main/main.c

```diff
@@ -1,13 +1,24 @@
 
-#include "nvsFlash.h"
-#include "freertos/FreeRTOS.h"
-#include "freertos/task.h"
 
 #include "lvglGui.h"
 
+#include "main.h"
+#include "nvsFlash.h"
+struct Token_ TokenParam;
+struct UserInfo_ UserInfo;
+
+QueueHandle_t BufQueue1;
+SemaphoreHandle_t GetResponseSemaphore = NULL;
+
+
+// wifi ssid is "Hardware10"  and wifi pass is "87654321"
+
+
 void app_main(void)
-{   
-    nvsFlashInit();    
+{
+    GlobalInit();
+    SpotifyComponent();
+    // nvsFlashInit();
```

Comment:
we should not permit to comment other auther's modules in main. we should run all of parts of project together